snippets/input-numbers.js

```diff
@@ -0,0 +1,20 @@
+/* This js snippet is used to add a phone number, social secruity number, or others*/
+
+var selector = "<TODO: REPLACE SELECTOR>"; /* replace with your selector here */
+var element = document.querySelector(selector);
+
+/* Stop process if element does not exist */
+if (!element) {
+    throw new Error('Error: cannot find the element with selector(' + selector + ').');
+}
+
+/* creating a function to change the phone number value */
+
+changeValue(element, "<REPLACE WITH NUMBER HERE>");
```

Comment:
It would be nice to turn "&lt;REPLACE WITH NUMBER HERE&gt;" into a variable and then declare it at the top